src/feature-gates.md

```diff
@@ -0,0 +1,65 @@
+# Feature Gates
+
+This chapter is intended to provide basic help for modifying feature gates. See
+["Stability in code"][stability-section] for help with *adding* feature gates.
```

Comment:
Could we have "stability in code" link here instead, so all the info is one place?

Reply:
Done. I thought of doing that before, but wasn't sure. Good suggestion!